app/helpers/date_time_helper.rb

```diff
@@ -4,7 +4,7 @@ module DateTimeHelper
   def format_date(date_str)
     begin
       date_formatted = date_str.to_date
-    rescue StandardError
+    rescue ArgumentError
```

Comment:
I think handling `ArgumentError` as well as `NoMethodError` here would be good, in case someone passes the wrong number of arguments or passes in an argument that doesn't support `.to_date`.